packages/fxa-settings/src/models/Account.ts

```diff
@@ -578,11 +593,26 @@ export class Account implements AccountData {
         `,
         variables: { input: { email } },
       });
-
       return result.data.passwordForgotSendCode;
     } catch (err) {
-      const errno = (err as ApolloError).graphQLErrors[0].extensions?.errno;
-      if (errno && AuthUiErrorNos[errno]) {
+      const graphQlError = (err as ApolloError).graphQLErrors[0];
```

Comment:
I added the `retryAfter`, `retryAfterLocalized` values to the thrown error here, but did not enable displaying these errors in the UI (currently, errors with resending links/codes are displayed as a generic error). 

@millsoper Enabling showing throttled error messages could be included in https://mozilla-hub.atlassian.net/browse/FXA-7147